examples/cloudflare-orange/app/env.d.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference types="./env.d.ts" />
```

Comment:
i've noticed that `.d.ts` doesn't type check and have been un-winding this. Better to rename to `.ts`

I am also not sure if the `/// reference` helps vs putting this in the tsconfig.json
```ts
{
  "compilerOptions": {
    "types": ["./app/env.ts"]
  }
}
```